### PR TITLE
[FAQs] Added FAQ in GTN for finding/working on shared histories

### PR DESCRIPTION
**Pull request by garimavs**
The following Pull Request is a part of the Issue: _Move FAQs to GTN_ of the [Galaxy_Hub repository](https://github.com/galaxyproject/galaxy-hub).

I have added [A history was shared with me, where can I find it?](https://galaxyproject.org/support/missing-history/) in the GTN.
Kindly have a look at it and suggest any improvements or amendments.

### CODE REVIEW

File: faqs/galaxy/histories_shared.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+**To find histories shared with me:**
+- Log into your account.
+- Select **User**, in the drop-down menu, select **Histories shared with me**.
+
+**To work with shared histories:**
+- Import the History into your account via copying it to work with it.
+- Unshare Histories that you no longer want shared with you or that you have already made a copy of.
+
+**Note:** Shared Histories (when copied into your account or not) do count in portion toward your total account data quota usage. More details on histories shared concerning account quota usage can be found in [this link](https://galaxyproject.org/support/account-quotas/#find-histories-that-have-been-shared-with-you-and-unshare-those-not-needed).
```
Comment (beatrizserrano) [inline faqs/galaxy/histories_shared.md:18]: maybe this link should be also another FAQ?

Comment (beatrizserrano) [reply faqs/galaxy/histories_shared.md:18]: The current link to galaxyproject.org will disappear soon, so the link would be broken. I think that I'd create a new FAQ and once it's merged, link it from here.

Comment (garimavs) [reply faqs/galaxy/histories_shared.md:18]: Okay, is there anything else that I can help with for this PR?

Comment (beatrizserrano) [reply faqs/galaxy/histories_shared.md:18]: I think it's good! Once the other FAQ is added, we can link it from here :)

Comment (beatrizserrano) [reply faqs/galaxy/histories_shared.md:18]: @adi-1999 are you planning to work on [this FAQ](https://galaxyproject.org/support/account-quotas/#find-histories-that-have-been-shared-with-you-and-unshare-those-not-needed), we need it to get this other one merged. Thanks!

Comment (garimavs) [reply faqs/galaxy/histories_shared.md:18]: In case @adi-1999 is unable to work on this, then I would like to volunteer for the task.

Comment (beatrizserrano) [reply faqs/galaxy/histories_shared.md:18]: Sure, thanks @garimavs! I've updated [the issue](https://github.com/galaxyproject/galaxy-hub/issues/1262).

**Comment by garimavs** [comment]
Hey @beatrizserrano, I have submitted a pull request regarding account quota usage [here](https://github.com/galaxyproject/training-material/pull/3372). Kindly have a look at it and suggest improvements.

File: faqs/galaxy/histories_shared.md (hunk 1 above)
Comment (garimavs) [reply faqs/galaxy/histories_shared.md]: Hey @beatrizserrano, I have updated the PR. Kindly have a look and suggest any further improvements or amendments.

File: faqs/galaxy/histories_shared.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+description: How to find and work on histories shared with you
+area: datasets
+layout: faq
+box_type: tip
+contributors: [jennaj, garimavs]
+---
+
+**To find histories shared with me:**
+- Log into your account.
+- Select **User**, in the drop-down menu, select **Histories shared with me**.
```
Comment (beatrizserrano) [inline faqs/galaxy/histories_shared.md:12]:
```suggestion
1. Log into your account.
2. Select **User**, in the drop-down menu, select **Histories shared with me**.
```

File: faqs/galaxy/histories_shared.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+**To work with shared histories:**
+- Import the History into your account via copying it to work with it.
+- Unshare Histories that you no longer want shared with you or that you have already made a copy of.
```
Comment (beatrizserrano) [inline faqs/galaxy/histories_shared.md:16]: are these also steps?

Comment (garimavs) [reply faqs/galaxy/histories_shared.md:16]: To work with shared histories, you need to copy them and then work on them, unsharing history part is more related to the account quota management thing.

File: faqs/galaxy/histories_shared.md (hunk 2 above)
Comment (beatrizserrano) [reply faqs/galaxy/histories_shared.md:12]: @garimavs does this suggestion make sense to you? I think this is the only pending change before merging

Comment (garimavs) [reply faqs/galaxy/histories_shared.md:12]: Sorry for the delay @beatrizserrano, this suggestion sounds good to me. Will update it shortly!

File: faqs/galaxy/histories_shared.md (hunk 1 above)
Comment (beatrizserrano) [reply faqs/galaxy/histories_shared.md:18]: this link is still pointing to the Hub, do we better link the FAQ?

Comment (garimavs) [reply faqs/galaxy/histories_shared.md:18]: Oh, I guess while editing the numbering, I edited the link twice. Sorry for the inconvenience, will undo it.

Comment (beatrizserrano) [reply faqs/galaxy/histories_shared.md]: cool, thank you! Ready to merge then! :)

**Review by beatrizserrano — APPROVED** [review]

**Pull request merged**